docs/runbooks/KubevirtHyperconvergedClusterOperatorNMOInUseAlert.md

```diff
@@ -0,0 +1,42 @@
+# KubevirtHyperconvergedClusterOperatorNMOInUseAlert
+
+## Meaning
+Starting from version 4.11.0, Node Maintenance Operator (NMO) is no longer included in OpenShift Virtualization.  
+This alert is being fired if both of the following conditions are met:
+- OpenShift version is 4.11 or higher.
+- The integrated NMO is being used, i.e. one or more custom resources of Kind `NodeMaintenance` and API group of `nodemaintenance.kubevirt.io` have been found on the cluster.
+
+Existence of such a custom resource indicates that the targeted node specified in `spec.nodeName` was put into a maintenance mode, which means the node is cordoned and server-side drained.
+
```

Comment:
@orenc1 you are missing the impact section

Reply:
Ok I moved the last sentence in the meaning section to an impact section.